### DISCUSSION TITLE
document translation [user_guides/tutorial 4: train and test in MMagic] 翻译文档【基础教程/教程4：训练与测试】

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#581] [MMagic]
# document translation [user_guides/tutorial 4: train and test in MMagic] 
 翻译文档【基础教程/教程4：训练与测试】 

translate docs/en/latest/user_guides/train_test.md into docs/zh_cn/latest/user_guides/train_test.md in Chinese

根据 docs/en/latest/user_guides/train_test.md 翻译 docs/zh_cn/latest/user_guides/train_test.md

## Task Information

**Repository**：MMagic

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：ththth888

**Claim Time**：2023年07月20日

**PR Link**：https://github.com/open-mmlab/mmagic/pull/1961

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by ththth888** [comment]
Claim the mission

**Reply by ththth888** [reply]
https://github.com/open-mmlab/mmagic/pull/1957#issue-1821481085

**Reply by crazysteeaam** [reply]
good job！

**Reply by ththth888** [reply]
This is the new PR I just made. Something goes wrong with the previous one. I think the current PR is now working. Here is the link.

**Reply by ththth888** [reply]
https://github.com/open-mmlab/mmagic/pull/1961
